docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.9"
+services:
+  monitor:
+    build:
+      context: .
+      dockerfile: Dockerfile.monitor
+    ports:
+      - "3001:3001"
+    volumes:
+      - ~/config:/oracle-cl-middleware/src/config
+      - ~/state:/oracle-cl-middleware/src/data
+    environment:
+      AGORIC_NET: agoriclocal 
+      AGORIC_RPC: http://$THIS_VM_IP:26657
+      EI_CHAINLINKURL: http://$THIS_VM_IP:6691
+      SERVICE: monitor
```

Comment:
I don't really like this variable and it's only being used for logging

Reply:
It's used so that when the monitoring script loads a function from a helper class which loads the middleware environment variable it does not fail